src/de/hasait/cipa/resource/CleanupResource.groovy

```diff
@@ -0,0 +1,12 @@
+package de.hasait.cipa.resource
+
+import de.hasait.cipa.CipaNode
+
+/**
+ * An interface to for performing clean up of resources.
+ */
+interface CleanupResource {
```

Comment:
A more intuitive name would be CipaResourceCleanup.
The implementation for CipaFileResources would be CipaFileResourceCleanup